tools/reactive-controllers/src/MatchMedia.ts

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import type { ReactiveController, ReactiveElement } from 'lit';
+
+export const DARK_MODE = '(prefers-color-scheme: dark)';
+export const IS_MOBILE =
```

Comment:
@davidyxu I've updated the query here to prevent the tray in desktop all together. This _should_ mitigate some of the question about wanting to switch from one to the other while open.

In general, I'd like not to switch between the two while open, do you have thoughts on how we could be more clear about that reality herein?

Reply:
👍 seems reasonable, not sure what the spectrum standards are but I wonder if just always using tray on phone vs always using popover for desktop/ipad would be the simplest world to reason about.

Maybe the best of all worlds would be some configurable attribute on sp-picker which defaults to responsive, and the documentation can describe what responsive behavior means here.

Reply:
Agree on this. Added to #2086 for tracking...